config/initializers/monkeypatches/item_viewer.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+# Overridden from:
+# https://github.com/geoblacklight/geoblacklight/blob/main/lib/geoblacklight/item_viewer.rb
```

Comment:
Question: Why'd we have to override this?

Reply:
Comments added.